eng/CredScanSuppression.json

```diff
@@ -87,12 +96,25 @@
                 "sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/SessionRecords/CertificateClientLiveTests/VerifyImportCertificatePem.json",
                 "sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/SessionRecords/CertificateClientLiveTests/VerifyImportCertificatePemAsync.json",
                 "sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/SessionRecords/CertificateClientLiveTests/VerifyImportCertificatePemWithoutIssuer.json",
-                "sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/SessionRecords/CertificateClientLiveTests/VerifyImportCertificatePemWithoutIssuerAsync.json"
-            ],
-            "_justification": "Files contain ephemeral keys or certificates used by test code during playback."
-        },
-        {
-            "file":[
+                "sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/SessionRecords/CertificateClientLiveTests/VerifyImportCertificatePemWithoutIssuerAsync.json",
```

Comment:
It is generally not recommended to add entire recording files to the ignore list as that opens up the door for new real secrets to sneak in. If the recording files cannot be fixed to contain a placeholder value or isolated to a secrets only file under suppressions then the recommendation is to baseline files so that if they change new secrets might be detected. See https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/623/Cred-scan-failure-troubleshooting for general guidance.

Reply:
Most of these record files are legacy, and recorded by service team which we can't fix, and the secret in these files are only for playback test which is not a real secret used by prod.

Reply:
I understand that the secrets in these files might not be live secrets anymore but by adding the entire files to the exclude list we are potentially missing live secrets in the future if these are updated.